src/System.Private.CoreLib/src/System/Threading/Lock.cs

```diff
@@ -9,13 +9,17 @@ namespace System.Threading
 {
     public sealed class Lock
     {
+        // The following constants define characteristics of spinning logic in the Lock class
+        const int SpinningNotInitialized = 0;
+        const int SpinningDisabled = -1;
+        const int MaxSpinningValue = 10000;
```

Comment:
I believe that the use of constants in cases like this is net loss - makes the code bigger and harder to understand.  But it depends on personal preference...